### PR TITLE
 list structure

### PR DESCRIPTION
**Pull request by Mhdez221993**
In this project, I build a simple HTML list of To-Do tasks, using:

1. Webpack to bundle JavaScript
2. ES6 syntax
3.  ES6 modules

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import './style.css';
+
+class TaskList {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedList = [{ description: 'Time to read', completed: false, index: 0 }, { description: 'Fix linters', completed: false, index: 1 }, { description: 'Do the homework', completed: false, index: 2 }];
```
Comment (mimipeshy) [inline src/index.js]: - [x] On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Mhdez221993 👋🏾 👋🏾 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- Good github workflow ✔️ 
- You have met most of the requirements needed for this milestone ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      checkbox.type = 'checkbox';
+      checkbox.className = 'checkbox';
+      const p = document.createElement('p');
+      const index = document.createElement('spand');
+      index.className = 'index';
+      const button = document.createElement('i');
+      button.classList.add('fa', 'fa-ellipsis-v', 'icon');
+
+      p.innerHTML = task.description;
+      index.innerHTML = task.index;
```
Comment (DeliceLydia) [inline src/index.js]: - [x] You are supposed to have an array of some simple to do tasks (array of objects) 
with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]
- [x] As the previous reviewer mentioned, on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task, currently when I load the page it doesn't show any task please work on that

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import './style.css';
+
+class TaskList {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedList = JSON.parse(localStorage.getItem('savedList')) || [];
+    // this.displayAllTask();
```
Comment (DeliceLydia) [inline src/index.js]: - [x] Remember to always remove commented code

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Mhdez221993 , 

Good job so far! it take a courage to reach this far.. but there are some issues that you still need to address to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (DeliceLydia) [reply src/index.js]: - [ ] Need to address this

File: src/index.js (hunk 2 above)
Comment (Mhdez221993) [reply src/index.js]: Thank you

File: src/index.js (hunk 3 above)
Comment (Mhdez221993) [reply src/index.js]: Done

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Mhdez221993, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**